src/net/RTP/RTPPacket.cs

```diff
@@ -18,19 +18,19 @@
 
 namespace SIPSorcery.Net
 {
-    public class RTPPacket
+    public sealed class RTPPacket
     {
         public RTPHeader Header;
         public byte[] Payload;
 
         public RTPPacket()
         {
-            Header = new RTPHeader();
+            Header = new RTPHeader(null);

```

Comment:
I'd rather have a constructor that takes no parameters thean require `null`.

Reply:
Right. Second that. I will reintroduce the original constructor, but leave it empty (no Crypto calls)